tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -12760,7 +12776,7 @@ write_extern_fndecl(struct LL_FnProto_ *proto)
         if (proto->abi->is_pure)
           print_token(" nounwind");
         if (proto->abi->is_pure)
-          print_token(" readnone");
+          print_token(" readnone willreturn");
```

Comment:
Can `"nounwind"` be added to this line too?

Reply:
yes